lib/src/services/yust_auth_service_dart.dart

```diff
@@ -91,6 +99,125 @@ class YustAuthService {
     throw UnsupportedError('Not supported. No UI available.');
   }
 
+  Future<YustUser?> createAccount(
+    String firstName,
+    String lastName,
+    String email,
+    String password, {
+    YustGender? gender,
+    bool useOAuth = false,
+  }) async {
+    GoogleCloudIdentitytoolkitV1SignUpResponse response;
+    final uuid = Uuid().v4();
+
+    // if (useOAuth == true) {
```

Comment:
remove comment & refactor authClient to yust.dart (use same one and extzend scopes)